### PR TITLE
Fix issues with Qt 5.15 where QSocketNotifier.activated became overloaded

### PR DESCRIPTION
**Pull request by lukas-hetzenecker**
After upgrading to Qt 5.15/PySide2 5.15.1 I noticed that the asyncio-Qt mainloop integration was completely broken - websockets stayed hanging in a connecting state, async HTTP requests never returned a response i.e. no callbacks were ever called.

Troubleshooting revealed that the `__on_read_activated` slots in `_unix.py` received a `PySide2.QtCore.QSocketDescriptor object`. Those were obviously not found in the `_fd_to_key` directory mapping, as the key expected there is the (int) file descriptor.

In the Qt documentation I found that the QSocketNotifier.activated behavior changed with Qt 5.15, whereas [previously](https://doc.qt.io/qt-5.14/qsocketnotifier.html#activated) only the `QSocketNotifier::activated(int socket)` signal existed, [now](https://doc.qt.io/qt-5.15/qsocketnotifier.html#activated) this signal became overloaded with `QSocketNotifier::activated(QSocketDescriptor socket, QSocketNotifier::Type type)` as arguments.

Long story short, this pull requests fixes the issue by explicitly connecting the signal with "int" soket fd as argument.

### CODE REVIEW

**Comment by lukas-hetzenecker** [comment]
Related merge request in asyncqt: gmarull/asyncqt#28

**Comment by TheGreatCabbage** [comment]
Nice work! I'll try and get a new release published tomorrow.

**Comment by TheGreatCabbage** [comment]
This is now live in version `0.9.5`. Thanks for your contribution!
